templates/repository/common/CONTRIBUTING.md

```diff
@@ -196,6 +196,8 @@ request, go through this checklist:
    command and confirm that it passes.
 1. Run `gofmt -s` (if the project is written in Go).
 1. Ensure that each commit has a subsystem prefix (ex: `controller:`).
```

Comment:
Maybe then "subsystem prefix" here is wrong? Because the prefixes in https://github.com/ory/hydra/blob/master/.github/semantic.yml are not about subsystems, but about a type of the commit. I think this whole bullet point should be reworded.

Reply:
Makes sense @mitar ! I will see to word this more accurately. 
Thanks for pointing it out.